panels/user-accounts/um-utils.c

```diff
@@ -460,24 +460,8 @@ down_arrow (GtkStyleContext *context,
         cairo_restore (cr);
 }
 
-static guint
-get_login_name_max (void)
-{
-#ifdef LOGIN_NAME_MAX
-        return LOGIN_NAME_MAX;
-#else
-        static gint length;
-
-        if (!length) {
-                length = sysconf (_SC_LOGIN_NAME_MAX);
-                g_assert_cmpint (length, >=, 0);
-        }
-
-        return length;
-#endif
-}
-
-#define MAXNAMELEN  get_login_name_max ()
+/* This is best hardcoded, see GNOME #766401. */
+#define MAXNAMELEN  32
```

Comment:
I see that in the upstream patch you took a different approach and still consult `sysconf`. Also Matthias had some comments on the patch. I would prefer if we followed upstream here, and if this is not really crucial to the patchset, I'd rather we branched it off in another issue (or use an upstream patch)